qutebrowser/utils/javascript.py

```diff
@@ -64,14 +64,28 @@ def _convert_js_arg(arg):
             arg, type(arg).__name__))
 
 
-def assemble(module, function, *args):
+def assemble(module, function, *args, delay=False):
     """Assemble a javascript file and a function call."""
     js_args = ', '.join(_convert_js_arg(arg) for arg in args)
     if module == 'window':
         parts = ['window', function]
     else:
         parts = ['window', '_qutebrowser', module, function]
-    code = '"use strict";\n{}({});'.format('.'.join(parts), js_args)
+    if delay:
```

Comment:
As you were able to find that this isn't a complete fix, we should probably revert this, and instead [fire an event](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/fireEvent) when the stylesheet loads, and hook into that if we load to early. I think that method is much more foolproof.

Reply:
You could do it that way or you could manually change the lines back. If you wanted you could rebase it away as well, whatever you're comfortable with :P.